RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc

```diff
@@ -29,14 +29,16 @@ ME0SegAlgoRU::ME0SegAlgoRU(const edm::ParameterSet& ps)
 
 	allowWideSegments = ps.getParameter<bool>("allowWideSegments");
 	doCollisions = ps.getParameter<bool>("doCollisions");
+
 	stdParameters.maxChi2Additional = ps.getParameter<double>("maxChi2Additional");
 	stdParameters.maxChi2Prune      = ps.getParameter<double>("maxChi2Prune"     );
 	stdParameters.maxChi2GoodSeg    = ps.getParameter<double>("maxChi2GoodSeg"   ) ;
 	stdParameters.maxPhiSeeds       = ps.getParameter<double>("maxPhiSeeds"      ) ;
 	stdParameters.maxPhiAdditional  = ps.getParameter<double>("maxPhiAdditional" );
 	stdParameters.maxETASeeds       = ps.getParameter<double>("maxETASeeds"      );
 	stdParameters.maxTOFDiff        = ps.getParameter<double>("maxTOFDiff"      );
-	stdParameters.minNumberOfHits        = ps.getParameter<unsigned int>("minNumberOfHits"      );
+	stdParameters.requireCentralBX  = ps.getParameter<bool>("requireCentralBX"      );
+	stdParameters.minNumberOfHits   = ps.getParameter<unsigned int>("minNumberOfHits"      );
 	stdParameters.requireBeamConstr = true;

```

Comment:
Thank you @kpedro88 for remembering me : indeed in that (already merged) PR we argued that in a next update of the code the hardcoded parameters should have been made configurable, and everything would have possibly moved to  ParameterSetDescription: aren't we arrived at that "next update" yet?